love/src/components/MainTel/M1M3TS/M1M3TS.jsx

```diff
@@ -0,0 +1,185 @@
+    /** Thermal status response data.  Measured fan RPM-0 to 2550 RPM. */
+    fanRPM: PropTypes.arrayOf(PropTypes.number),
+    /** Object with boolean arrays for indicates of warning. */
+    thermalWarnings: PropTypes.objectOf(PropTypes.arrayOf(PropTypes.bool)),
+    /** Mixing valve measured position. */
+    valvePosition: PropTypes.number,
+    /** Number of the minimum temperature limit, used for the gradiant color */
+    minTemperatureLimit: PropTypes.number,
+    /** Number of the maximum temerature limit, used for the gradiant color */
+    maxTemperatureLimit: PropTypes.number,
+  };
+
+  static defaultProps = {
+    summaryState: 1,
+    fanHeaters: false,
+    coolantPump: false,
+    setpoint: 0,
+    ilcFCU: [],
+    // enabledFCU: [],
+    absoluteTemperature: [],
+    differentialTemperature: [],
+    fanRPM: [],
+    thermalWarnings: [],
+    valvePosition: 0,
+    minTemperatureLimit: 0,
+    maxTemperatureLimit: 1000,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      showFcuIDs: true,
+      showDifferentialTemp: true,
+      showWarnings: true,
+      selectedSensor: null,
+    };
+  }
+
+  componentDidMount() {
+    this.props.subscribeToStreams();
+  }
+
+  componentWillUnmount() {
+    this.props.unsubscribeToStreams();
+  }
+
+  arrayReferenceId = () => {
+    const array = [];
+    for (let i = 0; i < 96; i++) {
+      array.push(i + 1);
+    }
+    return array;
+  };
+
+  render() {
+    // Summary
+    const { summaryState, fanHeaters, coolantPump } = this.props;
+
+    // Temperature
+    const { setpoint, minTemperatureLimit, maxTemperatureLimit } = this.props;
+
+    // Info
+    const {
+      referenceId,
+      ilcFCU,
+      // enabledFCU,
+      absoluteTemperature,
+      differentialTemperature,
+      fanRPM,
+      fanBreaker,
+      heaterDisabled,
+    } = this.props;
+
+    const { thermalWarnings } = this.props;
+
+    // Mixing
+    const { valvePosition } = this.props;
+
+    return (
+      <div className={styles.container}>
+        <div className={styles.summaryContainer}>
+          <Summary summaryState={summaryState} fanHeaters={fanHeaters} coolantPump={coolantPump} />
+        </div>
+
+        <div className={styles.separatorContainer}>
+          <div className={styles.separator}></div>
+        </div>
+
+        <div className={styles.menuContainer}>
+          <Menu
+            showFcuIDs={this.state.showFcuIDs}
+            showDifferentialTemp={this.state.showDifferentialTemp}
+            showWarnings={this.state.showWarnings}
+            toggleFcuIDs={(show) => this.setState({ showFcuIDs: show })}
+            toggleTemperature={(show) => this.setState({ showDifferentialTemp: show })}
+            toggleWarnings={(show) => this.setState({ showWarnings: show })}
+          />
+        </div>
+
+        <div className={styles.selectorContainer}>
+          <Selector
+            sensorReferenceId={referenceId}
+            // enabled={enabledFCU}
```

Comment:
Will this variable be used? If not, maybe it should be removed.

Reply:
Same as stated above.